CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
While it's nice to have this file autogenerated, I don't think its current form is helpful for users, I'd rather take this and rewrite the entries manually, mainly to combine entires meaningfully.

Reply:
I understand that you will rework the `CHANGELOG.md`? Great 👍